client/src/index.js

```diff
@@ -77,7 +77,14 @@ window.addEventListener('load', async() => {
   if(window.web3) {
 
     ethutil.setWeb3(window.web3)
-    ethutil.attachLogger()
+
+    try {
```

Comment:
Now that `attachLogger` has logic for handling a non-metamask provider I would just revert the changes to this file. The error will get printed anyway.